### ISSUE TITLE
Login/Register & Purchase page: Hidden "Options" not focusable 

### ISSUE THREAD

**Issue by swickr** [w3c/validator-suite#183]
The "Options" available for a chosen job type are not accessible to a screen reader user. Because of the "forms mode" mentioned above, the link to expose the options is not in the tab order. Consider adding a <code>tabindex</code> of 0 to these elements to make them focusable. Also, add aria that indicates current state of the widget (expanded or collapsed). Another solution would be to add an <code>href</code> attribute to the anchor, which would make it focusable. Hidden text (offscreen) would be required to explain the behvior ("toggle the options open"). Another solution would be to use the <code>&lt;summary&gt;</code> and <code>&lt;details&gt;</code> elements which was specifically designed for this situation.

In [Mark's full assessment](https://lists.w3.org/Archives/Team/team-vs/2013Aug/att-0010/ValidatorSuiteA11y.html) he adds

> _Note: &lt;summary&gt; and &lt;details&gt; is not currently implemented in all browsers and use of a polyfill would be required._
